### PR TITLE
Routing Improvements

### PR DESCRIPTION
**Pull request by kkopanidis**
The target of this PR is to make routing declaration more consistent and clear older code and methods of declaring routes.
Goals:
- Consistent declaration and features between Admin and Client routers
- Remove custom code from Admin/Client router grpc client files
- Create any and all utility functions required to add enough sugar in route declaration

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ x Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:
Existing applications should use the new functions introduced and remove the older functions. Applications using the 0.11 sdk version should not be affected.

**The PR fulfills these requirements:**

- [x] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by kkopanidis** [comment]
I'm still thinking that maybe I need to also make the admin changes in this PR, so that admin and client router registrations are consistent but I'll leave it to the reviewers to decide if the PR is already too big

File: modules/forms/src/controllers/forms.controller.ts [hunk 1]
```diff
@@ -36,10 +35,10 @@ export class FormsController {
       });
   }
 
-  refreshRoutes() {
+  private async loadExistingForms() {
```
Comment (kon14) [inline modules/forms/src/controllers/forms.controller.ts:38]: Why do we need a private copy of refreshRoutes?

**Review by kon14 — CHANGES_REQUESTED** [review]
Can we keep a consistent naming for RoutingManager object members?
Some classes use _routingController while others use _routingManager.

Besides that, I think we can get rid of `loadExistingForms()` in `Forms.FormsController` and call the public `refreshRoutes` in its constructor just like we do everywhere else.

Beyond that, looks LGTM 🎉

**Comment by kon14** [comment]
> I'm still thinking that maybe I need to also make the admin changes in this PR, so that admin and client router registrations are consistent but I'll leave it to the reviewers to decide if the PR is already too big

Lets address that in a followup PR. Prettier already produces huge diffs.

**Review by kon14 — APPROVED** [review]

**Review by sdimitris — APPROVED** [review]

**Review by Michael-Vol — APPROVED** [review]

**Pull request merged**